test/bad_dtls_test.c

```diff
@@ -329,11 +330,12 @@ static int send_record(BIO *rbio, unsigned char type, uint64_t seqnr,
     } while (len % 16);
 
     /* Generate IV, and encrypt */
-    RAND_bytes(iv, sizeof(iv));
-    enc_ctx = EVP_CIPHER_CTX_new();
-    EVP_CipherInit_ex(enc_ctx, EVP_aes_128_cbc(), NULL, enc_key, iv, 1);
-    EVP_Cipher(enc_ctx, enc, enc, len);
-    EVP_CIPHER_CTX_free(enc_ctx);
+    if (!TEST_true(RAND_bytes(iv, sizeof(iv)))
```

Comment:
Above there are unchecked calls to EVP_MAC_... functions I suppose they'll be marked by Coverity later as well.

Reply:
If enough callers check the result, they will be flagged.

I'll add some extra checks.